templates/guides/getting-started.md.tmpl

```diff
@@ -41,22 +41,40 @@ provider "qdrant-cloud" {
 ### Creating your first Cluster (including access token):
```

Comment:
I know you haven't updated this part in the PR, but might be worth to update this sentence. Because we don't refer to `access token` in our product, doing here can be confusing.